skimage/util/process.py

```diff
@@ -0,0 +1,68 @@
+from multiprocessing import cpu_count
+
+import dask.array as da
+
+__all__ = ['process_chunks']
+
+
+def _get_chunks(shape, ncpu):
```

Comment:
Thanks for this! I am realising that I didn't understand the chunk specification format... And I still don't! I was indeed thinking about block sizes, as @mrocklin suggests. (h5py, for example, specifies chunking in this way.) Can you explain the format a bit more? Maybe a few examples of how a chunk tuple gets unpacked to array indices? This could go in a Notes section for the `process_chunks` function docstring. I don't imagine I'll be the first to give something completely different to what's expected!